persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_19/entities.bal

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/persist;
+
+@persist:Entity {
+    key: ["id"],
+    tableName: "Profiles"
+}
+public type Profile record  {|
+    readonly int id;
+    string name;
+    @persist:Relation {reference: ["id"]}
+    User user?;
+|};
+
+@persist:Entity {
+    key: ["id"],
+    tableName: "Users"
+}
+public type User record  {|
+    readonly int id;
+    string name;
+    Profile profile?;
+|};
+
+@persist:Entity {
+    key: ["id"],
+    tableName: "MultipleAssociations"
+}
+public type MultipleAssociations record {|
+    readonly int id;
+    string name;
+
+    @persist:Relation {keyColumns: ["profileId"]}
+    Profile profile?;
+
+    @persist:Relation
+    User user?;
+|};
+
+
+
```

Comment:
```suggestion
```

Reply:
remove additional new lines in other places

Reply:
Updated in [c7709b2](https://github.com/ballerina-platform/persist-tools/pull/48/commits/c7709b28a97d8f91b708b4bd3d483e0a0eebb499)